CMakeLists.txt

```diff
@@ -371,18 +371,9 @@ configure_file(${EXT_CONTROL_FILE}.in ${EXT_CONTROL_FILE})
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${EXT_CONTROL_FILE}
         DESTINATION "${PG_SHAREDIR}/extension")
 
-# We look for specific versions installed by distributions before the default
-# name since distros that have installed clang-format-9 will not work and the
-# user need to install an earlier version, which will then be named
-# "clang-format-N".
-#
-# This breaks the CMake convention of using the "default" name first to handle
-# local installs. If this turns out to be something that we want to support, we
-# need to look specifically for "clang-format" in the local installation paths
-# before looking for the versioned names in standard installation paths.
 find_program(
   CLANG_FORMAT
-  NAMES clang-format-8 clang-format-7 clang-format
+  NAMES clang-format
```

Comment:
The `-Wno-error=unknown` gives us a modicum of backwards compatibility, so I think we can just use _the_ clang-format that is present on the machine, without preferring a particular version. Or should we still prefer 14?